FreeRTOS/iot-reference-arm-corstone3xx
# Update ml-kit integration

Update ml-kit integration

Description
-----------
Use ML Embedded Evaluation kit as a collection of libraries, instead of building it as an external project.
Update ML-kit version to 23.11

Test Steps
-----------
Compiled and run the available examples with GCC and Armclang for Corstone-300 and 310.

Checklist:
----------


- [] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
